osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs

```diff
@@ -470,7 +470,7 @@ protected override bool OnMouseDown(MouseDownEvent e)
             {
                 marker.Active = true;
                 handleMouseInput(e.ScreenSpaceMousePosition);
-                return base.OnMouseDown(e);
+                return true;
```

Comment:
Here I am wondering why this didn't work. It's another case of nested classes.

I really think we should stop nesting classes. It's very hard to deal with as a developer.